### DISCUSSION TITLE
Unable to Set Cron Jobs

### DISCUSSION THREAD

**Discussion by sobeservices** [alexjustesen/speedtest-tracker#1634] [Q&A] [answered]
What happen to the cron jobs section. I got the new 20.8 up and running but its not running automatic test like before and I see no where to set the cron's. Before I belivew it was a gear icon that said general and you can set cron's where did it go?

**Comment by svenvg93** [comment, marked as answer]
As of version 0.20.x the settings page has been removed for environment variables. Please read https://github.com/alexjustesen/speedtest-tracker/discussions/1470 on how to set the cron jobs now :)
